.env.example

```
@@ -1,21 +1,33 @@
+PORT=4000
+CF_ACCOUNT_ID=YOUR-CLOUDFLARE-ACCOUNT-ID
+CF_TURN_KEY_ID=YOUR-CLOUDFLARE-TURN-KEY-ID
+CF_TURN_API_TOKEN=YOUR-CLOUDFLARE-TURN-API-TOKEN
+R2_ACCESS_KEY_ID=YOUR-R2-ACCESS-KEY-ID
+R2_ACCOUNT_ID=YOUR-R2-ACCOUNT-ID
+R2_BUCKET_NAME=YOUR-R2-BUCKET-NAME
+R2_PUBLIC_DOMAIN=YOUR-R2-PUBLIC-DOMAIN
+R2_SECRET_ACCESS_KEY=YOUR-R2-SECRET-ACCESS-KEY
+REDIS_URL="redis://127.0.0.1:6379"
+TURNSTILE_SECRET_KEY=YOUR-TURNSTILE-SECRET-KEY
+UPLOAD_DIR=uploads // optional, default is "uploads"
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)